lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
You can test this internally on a devserver where you can ssh as a normal user or root. Swing by my desk if you want some help.